python/cudf/cudf/tests/test_udf_masked_ops.py

```diff
@@ -738,214 +743,172 @@ def f(x):
     assert precompiled.currsize == 1
 
 
-@string_udf_test
-def test_string_udf_len(str_udf_data):
-    def func(row):
-        return len(row["str_col"])
-
-    run_masked_udf_test(func, str_udf_data, check_dtype=False)
-
-
-@string_udf_test
-def test_string_udf_startswith(str_udf_data, substr):
-    def func(row):
-        return row["str_col"].startswith(substr)
-
-    run_masked_udf_test(func, str_udf_data, check_dtype=False)
-
-
-@string_udf_test
-def test_string_udf_endswith(str_udf_data, substr):
-    def func(row):
-        return row["str_col"].endswith(substr)
-
-    run_masked_udf_test(func, str_udf_data, check_dtype=False)
-
-
-@string_udf_test
-def test_string_udf_find(str_udf_data, substr):
-    def func(row):
-        return row["str_col"].find(substr)
-
-    run_masked_udf_test(func, str_udf_data, check_dtype=False)
-
-
-@string_udf_test
-def test_string_udf_rfind(str_udf_data, substr):
-    def func(row):
-        return row["str_col"].rfind(substr)
-
-    run_masked_udf_test(func, str_udf_data, check_dtype=False)
-
-
-@string_udf_test
-def test_string_udf_contains(str_udf_data, substr):
-    def func(row):
-        return substr in row["str_col"]
-
-    run_masked_udf_test(func, str_udf_data, check_dtype=False)
-
-
-@string_udf_test
-@pytest.mark.parametrize("other", ["cudf", "123", "", " "])
-@pytest.mark.parametrize("cmpop", comparison_ops)
-def test_string_udf_cmpops(str_udf_data, other, cmpop):
-    def func(row):
-        return cmpop(row["str_col"], other)
-
-    run_masked_udf_test(func, str_udf_data, check_dtype=False)
-
-
-@string_udf_test
-def test_string_udf_isalnum(str_udf_data):
-    def func(row):
-        return row["str_col"].isalnum()
-
-    run_masked_udf_test(func, str_udf_data, check_dtype=False)
-
+@pytest.mark.parametrize(
+    "data", [[1.0, 0.0, 1.5], [1, 0, 2], [True, False, True]]
+)
+@pytest.mark.parametrize("operator", [float, int, bool])
+def test_masked_udf_casting(operator, data):
+    data = cudf.Series(data)
 
-@string_udf_test
-def test_string_udf_isalpha(str_udf_data):
-    def func(row):
-        return row["str_col"].isalpha()
+    def func(x):
+        return operator(x)
 
-    run_masked_udf_test(func, str_udf_data, check_dtype=False)
+    run_masked_udf_series(func, data, check_dtype=False)
 
 
-@string_udf_test
```

Comment:
All of the test of this file is whitespace change, I grouped the string udf tests into a class to make skipping them easier.